Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/SharedViewBookmarkComponent.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzToolsFramework/ToolsComponents/EditorComponentBase.h>
+#include <Viewport/ViewBookmarkLoaderInterface.h>
+
+namespace AzToolsFramework
+{
+    //! @class EditorViewBookmarks
+    //! @brief struct that stores a vector of View bookmarks
```

Comment:
nit: It's customary to include a period (`.`) at the end of each line to indicate the end of the API comment (in some cases Doxygen does need this)